### PR TITLE
Set LICENSE_EDITION to cloudEdition

### PR DESCRIPTION
**Pull request by tmjd**
## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: config/enterprise_versions.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ title: v3.5.0
 components:
   cnx-manager:
     image: tigera/cnx-manager
-    version: v3.5.0
+    version: v3.5.1-1
```
Comment (doublek) [inline config/enterprise_versions.yml:5]: Should this say `tesla-`?

Comment (tmjd) [reply config/enterprise_versions.yml:5]: The gen-versions includes an update to add on the `tesla-`. That way when we merge in the future this branch will have the tag updated to match.

Comment (tmjd) [reply config/enterprise_versions.yml:5]: That file is used to generate a go file that is actually used, it is in the generation where the `tesla-` is added.

**Review by doublek — APPROVED** [review]
LGTM

**Review by ozdanborne — APPROVED** [review]

**Pull request merged**